src/queries/fei-goals.sql

```diff
@@ -0,0 +1,107 @@
+/**
+* This query collects all the FEI and near FEI goals based on several curiteriea.
+*
+* The query results are filterable by the JDI flags. All JDI flags are passed as an array of values
+* The following are the available flags within this script:
+* - jdi.regionIds - one or more values for 1 through 12
+* - jdi.recipients - one or more verbatium recipient names
+* - jdi.grantNumber - one or more verbatium grant numbers
+* - jdi.goal - one or more verbatium goal text
+* - jdi.status - one or more verbatium statuses
+* - jdi.createdVia - one or more verbatium created via values
+* - jdi.onApprovedAR - true or false
+* - jdi.response - one or more verbatium response values
+* - jdi.createdbetween - two dates defining a range for the createdAt to be within
+*
+* zero or more JDI flags can be set within the same transaction as the query is executed.
+* The following is an example of how to set a JDI flag:
+* SELECT SET_CONFIG('jdi.createdbetween','["2023-10-01","2023-10-15"]',TRUE);
+*/
+with bad as (
+	SELECT *
+	FROM public."Goals"
+	where name != '(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)'
+	AND name like '%monthly reported enrollment%'
+	and COALESCE("goalTemplateId",0) != 19017
+	AND id not in (52460, 52461, 52462, 52463, 52248, 52251, 52249, 52250, 55244, 55172, 55908, 55420, 55421, 56033, 50565, 50613, 50612, 50343, 50614)
+	UNION
+	SELECT *
+	FROM public."Goals"
+	where name = '(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)'
+	and COALESCE("goalTemplateId",0) != 19017
+	UNION
+	SELECT *
+	FROM public."Goals"
+	where name != '(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)'
+	AND name like '(FEI)%'
+	and COALESCE("goalTemplateId",0) != 19017
+	UNION
+	SELECT *
+	FROM  public."Goals"
+	WHERE "goalTemplateId" = 19017
+		)
+		select
+			r.name "recipient",
+			gr.number "grant number",
+			gr."regionId",
+			b.id "goalId",
+			b."goalTemplateId",
+			b.name "goal text",
+			b."createdAt",
+			b.status,
+			b."createdVia",
+			b."onApprovedAR",
+			gfr.response
```

Comment:
I take it that historically it isn't a problem that this column is "Response" when the ticket is asking for "Root cause"

Reply:
"response" is the general term for a response to a question prompt associated with a goal template. Currently we only have the FEI question about root causes, but we could have other questions in the future

Reply:
I understand that, more asking about the expectations of the people using this CSV

Reply:
But yeah, if we've been using this SQL to generate a CSV and no one has cared, I'm sure it's fine